frontend/src/pages/agreements/review/ReviewAgreement.jsx

```diff
@@ -109,9 +108,29 @@ export const ReviewAgreement = () => {
 
     const handleSendToApproval = () => {
         if (anyBudgetLinesDraft || anyBudgetLinePlanned) {
-            //Create BLI Package, and send it to approval (create a Workflow)
-            const bli_ids = getSelectedBudgetLines(budgetLines).map((bli) => bli.id);
-            const user_id = activeUser?.id;
+            const selectedBudgetLines = getSelectedBudgetLines(budgetLines);
+            let selectedBLIsWithStatusAndNotes = [];
+
+            switch (action) {
+                case actionOptions.CHANGE_DRAFT_TO_PLANNED:
+                    selectedBLIsWithStatusAndNotes = selectedBudgetLines.map((bli) => {
+                        return { id: bli.id, status: BLI_STATUS.PLANNED, requestor_notes: notes };
+                    });
```

Comment:
paired down to only `id, status, and notes`
- thx @stevtek for the help with this